source/content/guides/decoupled/wp-nextjs-frontend-starters/03-caching.md

```diff
@@ -0,0 +1,54 @@
+---
+title: WordPress + Next.js Frontend Starter for Front-End Sites
+subtitle: Purge Surrogate Key-based Cache
```

Comment:
Generally have the same comments as the Drupal entry - think we need more context with deeper sub-sections on Edge Caching, and then a related section on Purging the edge cache.